packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -30,7 +34,22 @@ export interface MenuItemProps<T = any> {
 /** @internal */
 export const MenuItem = React.memo(
   React.forwardRef<HTMLAnchorElement & HTMLButtonElement, MenuItemProps>(
-    ({ url, icon, label, ariaLabel, target, onClick, className, active, tabIndex = -1 }, ref) => {
+    (
+      {
+        url,
+        icon,
+        label,
+        ariaLabel,
+        ariaChecked,
+        target,
+        onClick,
+        className,
+        active,
+        role = 'menuitem',
+        tabIndex = -1,
+      },
```

Comment:
This has gotten a bit out of hand and tricky to read - maybe lets just get this variable out the single `props` object, and then unpack it down in the body of the function?


```ts
(props, ref) => {
  const { url, icon, etc ... } props;
  const styles = useStyles2(getStyles);
```